packages/core/services/google-maps-types.ts

```diff
@@ -82,6 +82,15 @@ export interface MarkerLabel {
   text: string;
 }
 
+export interface MarkerIcon {
+  anchor?: any | Point;
```

Comment:
why are all of these typed `any | Something`?

Reply:
That's because interface props without `?` causes compile error.
For example, `Size` interface checks every prop exists even if I just want to use width & height
```
export interface Size {
    height: number;
    width: number;
    equals(other: Size): boolean;
    toString(): string;
}
```
I wanted to add question mark on every props of `Size`, but I just picked other solution.

Reply:
because Size is a class, not a literal interface. Google Maps API doesn't just support  `{width: 100, height: 200}` .  You're supposed to use their provided class with constructor. https://developers.google.com/maps/documentation/javascript/reference/coordinates#Size

Reply:
Got it, I was using it in a wrong way